.gitignore

```diff
@@ -1,4 +1,8 @@
-.build
+# Common build output directory names
+.build/
```

Comment:
Do we still need this if we're using `build-output/` now?

Reply:
I would prefer to, yes, I use that directory name.